src/components/App/index.js

```diff
@@ -8,19 +8,47 @@ import JoinGame from '../JoinGame';
 import Lobby from '../Lobby';
 import Main from '../Main';
 import ErrorScreen from '../ErrorScreen';
+import Cable from 'actioncable';
+
 
 export class App extends Component {
   constructor(){
     super()
     this.state={
-      socket: "0000"
+      token: "000"
     }
   }
 
-  handleWebSocket = ({name,value}) => {
+  handleUserInit = ({name,token}) => {
+    console.log(token)
    this.setState({
-     socket:value
-   })
+     token
+   }, () => this.createCable(token) )
+   
+  }
+  createCable = (token) => {
+    if(this.state.token === token ){
+      let cable = Cable.createConsumer(`ws://localhost:3000/cable/${token}`);
```

Comment:
Will this path change when we have it fully deployed to Heroku?

Reply:
The localhost part of it will yes, not the cable and token however